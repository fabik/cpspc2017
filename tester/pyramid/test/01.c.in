7 50
1724819724 3 14316 4 2 10 29
244196
205064
594678
895409
931051
349704
686482
863854
654619
203537
60712
320395
572224
225327
134705
135703
77628
291450
502671
499938
36510
808451
495366
449994
564417
279208
240790
455250
258014
153238
851593
120415
72171
588694
295923
685207
634123
137229
857201
485284
322332
135040
54749
787067
484532
560590
407253
522419
813587
330600
