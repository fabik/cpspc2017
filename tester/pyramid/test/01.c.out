360086
302383
876898
1320348
1372905
515666
1012270
1273820
965286
300130
89524
472446
843788
332262
198633
200104
114468
429765
741226
737198
53836
1192122
730455
663550
832275
411713
355064
671301
380462
225962
1255738
177561
106422
868074
436362
1010390
935062
202354
1264008
715588
475304
199126
80732
1160590
714478
826634
600525
770346
1199697
487496
